### ISSUE TITLE
Error installing tailscale-install.timer -- file exists

### ISSUE THREAD

**Issue by berry** [SierraSoftworks/tailscale-udm#82]
Tried to re-install Tailscale on UDM SE but I always get an error installing the tailscale-install.timer. See:

```
root@Dream-Machine-Special-Edition:~# systemctl stop tailscaled
root@Dream-Machine-Special-Edition:~# rm -Rf /data/tailscale
root@Dream-Machine-Special-Edition:~# # Install the latest version of Tailscale UDM
curl -sSLq https://raw.github.com/SierraSoftworks/tailscale-udm/main/install.sh | sh
Installing latest Tailscale package repository...
Updating package lists...
Hit:1 https://deb.debian.org/debian bullseye InRelease
Hit:2 https://deb.debian.org/debian bullseye-updates InRelease
Hit:3 https://security.debian.org/debian-security bullseye-security InRelease
Hit:4 https://deb.debian.org/debian bullseye-backports InRelease
Hit:5 https://apt.artifacts.ui.com bullseye InRelease
Get:6 https://pkgs.tailscale.com/stable/debian bullseye InRelease
Fetched 6566 B in 1s (6023 B/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
36 packages can be upgraded. Run 'apt list --upgradable' to see them.
Installing Tailscale 1.60.0...
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
tailscale is already the newest version (1.60.0).
0 upgraded, 0 newly installed, 0 to remove and 36 not upgraded.
Configuring Tailscale port...
Configuring Tailscaled startup flags...
Restarting Tailscale daemon to detect new configuration...
Enabling Tailscale to start on boot...
Installing auto-update timer to ensure that Tailscale is kept installed and up to date.
ln: failed to create symbolic link '/etc/systemd/system/tailscale-install.timer': File exists
```

Using the instructions in the 2.5.2 update, seem results:

```
root@Dream-Machine-Special-Edition:~# systemctl disable tailscale-install.service
Removed /etc/systemd/system/multi-user.target.wants/tailscale-install.service.
Removed /etc/systemd/system/tailscale-install.service.
root@Dream-Machine-Special-Edition:~# /data/tailscale/manage.sh install!
Installing latest Tailscale package repository...
Updating package lists...
Hit:1 https://deb.debian.org/debian bullseye InRelease
Hit:2 https://deb.debian.org/debian bullseye-updates InRelease
Hit:3 https://security.debian.org/debian-security bullseye-security InRelease
Hit:4 https://deb.debian.org/debian bullseye-backports InRelease
Hit:5 https://apt.artifacts.ui.com bullseye InRelease
Get:6 https://pkgs.tailscale.com/stable/debian bullseye InRelease
Fetched 6566 B in 1s (6105 B/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
36 packages can be upgraded. Run 'apt list --upgradable' to see them.
Installing Tailscale 1.60.0...
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
tailscale is already the newest version (1.60.0).
0 upgraded, 0 newly installed, 0 to remove and 36 not upgraded.
Configuring Tailscale port...
Configuring Tailscaled startup flags...
Restarting Tailscale daemon to detect new configuration...
Enabling Tailscale to start on boot...
Installing pre-start script to install Tailscale on firmware updates.
Created symlink /etc/systemd/system/multi-user.target.wants/tailscale-install.service → /data/tailscale/tailscale-install.service.
Installing auto-update timer to ensure that Tailscale is kept installed and up to date.
ln: failed to create symbolic link '/etc/systemd/system/tailscale-install.timer': File exists
```

I tried several reinstalling, removing tailscale and then install. Nothing seems to work. My hope was that this timer would fixed another issue I have. Tailscale stops working now and then. My suspicion is after an update. So, my hope was the 2.5.2 update would solve this :-)

**Comment by notheotherben** [comment]
Thanks for letting me know, I think the issue is a broken symlink (as there's already a test for the file existing). I've released [v2.5.3](https://github.com/SierraSoftworks/tailscale-udm/releases/tag/v2.5.3) with some better handling for this scenario if you'd like to try that out.

**Comment by berry** [comment]
Installation ran without error. It seems to work. Thank you very much!

```
root@Dream-Machine-Special-Edition:~# curl -sSLq https://raw.github.com/SierraSoftworks/tailscale-udm/main/install.sh | sh
Tailscale is already installed and running, if you wish to update it, run '/data/tailscale/manage.sh update'
If you wish to force a reinstall, run '/data/tailscale/manage.sh install!'
Tailscaled started successfully
Run tailscale up to configure the interface.
root@Dream-Machine-Special-Edition:~# systemctl disable tailscale-install.service
Removed /etc/systemd/system/multi-user.target.wants/tailscale-install.service.
Removed /etc/systemd/system/tailscale-install.service.
root@Dream-Machine-Special-Edition:~# /data/tailscale/manage.sh install!
Installing latest Tailscale package repository...
Updating package lists...
Hit:1 https://security.debian.org/debian-security bullseye-security InRelease
Hit:2 https://deb.debian.org/debian bullseye InRelease
Get:3 https://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]
Get:4 https://deb.debian.org/debian bullseye-backports InRelease [49.0 kB]
Get:5 https://apt.artifacts.ui.com bullseye InRelease [13.0 kB]
Get:6 https://pkgs.tailscale.com/stable/debian bullseye InRelease
Get:7 https://apt.artifacts.ui.com bullseye/release arm64 Packages [43.6 kB]
Fetched 156 kB in 1s (137 kB/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
36 packages can be upgraded. Run 'apt list --upgradable' to see them.
Installing Tailscale 1.60.0...
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
tailscale is already the newest version (1.60.0).
0 upgraded, 0 newly installed, 0 to remove and 36 not upgraded.
Configuring Tailscale port...
Configuring Tailscaled startup flags...
Restarting Tailscale daemon to detect new configuration...
Enabling Tailscale to start on boot...
Installing pre-start script to install Tailscale on firmware updates.
Created symlink /etc/systemd/system/multi-user.target.wants/tailscale-install.service → /data/tailscale/tailscale-install.service.
Installation complete, run '/data/tailscale/manage.sh start' to start Tailscale
Installation complete, run '/data/tailscale/manage.sh start' to start Tailscale
root@Dream-Machine-Special-Edition:~# /data/tailscale/manage.sh start
Tailscaled started successfully
Run tailscale up to configure the interface.
```
